build/astump-overrides.properties

```diff
@@ -0,0 +1,50 @@
+STAREXEC_APPNAME: starexecfiolic

```

Comment:
This file should not be in the repo.  I have removed it from the branch.  Let us please not add it by accident again.  (I removed it a couple times now.)